gitops/generate.go

```diff
@@ -90,6 +91,23 @@ func Generate(fs afero.Afero, gitOpsFolder string, outputFolder string, componen
 
 // GenerateOverlays generates the overlays dir from a given BindingComponent
 func GenerateOverlays(fs afero.Afero, gitOpsFolder string, outputFolder string, component appstudioshared.BindingComponent, imageName, namespace string, componentGeneratedResources map[string][]string) error {
```

Comment:
I guess what I mean is, why aren't custom patches in the base folder supported?

Reply:
emm. I also have the same question when I was checking the design doc. looks like only overlays folder is expected to have custom patches provided. @maysunfaisal do you guys have any discussion on that with HACBS team?

Reply:
Since it's part of the phase 2 design, I won't block the PR on leaving it out, but it would be worthwhile having a discussion on it 🤔

Reply:
Yeah I wasnt aware of base dir having custom patches. It was more targeted towards an environment. 

I guess if a user really want to have some sort of base patch for all the environment, then they should put those configurations in the component level from the UI.. and it will get reflected in the base dir generation

Reply:
Let's leave as is for now, and can discuss later if we need/want to support patches under the base folder?